packages/block-library/src/image/edit.native.js

```diff
@@ -187,6 +191,14 @@ class ImageEdit extends React.Component {
 		}
 	}
 
+	getIcon( isRetryIcon ) {
+		if ( isRetryIcon ) {
+			return <SvgIconRetry fill={ styles.iconRetry.fill } />;
+		}
+
+		return <SvgIcon fill={ styles.icon.fill } />;
```

Comment:
Hey @Tug 👋 I updated icon js files to allow injecting props. Tested with web also, don't seem to have a side effect.  How does this solution look to you?

Reply:
I did a quick test to show what it would look like and pushed here. But after discussing it with you on another channel we agreed that it would be best to handle this separately.